src/uct/ib/base/ib_iface.h

```diff
@@ -196,6 +196,7 @@ typedef struct uct_ib_iface_init_attr {
     unsigned    fc_req_size;            /* Flow control request size */
     int         qp_type;                /* IB QP type */
     int         flags;                  /* Various flags (see enum) */
+    unsigned    max_rd_atomic;
```

Comment:
add description - `The maximum number of outstanding Atomic/RDMA reads allowed on SQ/RQ`?